orchestration/dagster_orchestration/pytest.ini

```diff
@@ -2,3 +2,4 @@
 console_output_style = progress
 env_files =
 	.env.test
+markers = e2e: marks tests as an end-to-end test (deselect with '-m "not e2e"')
```

Comment:
I've added this marker so we can segment out potentially slow-running E2E tests and not have them run locally if we so choose. In reality this is actually quick (~20 seconds), so it should be ok to keep in our default local test suite.